<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutomatedRssDownloader</class>
 <widget class="QDialog" name="AutomatedRssDownloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RSS Downloader</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="labelWarn">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;</string>
     </property>
     <property name="text">
      <string>Auto downloading of RSS torrents is currently disabled. You can enable it in application settings.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QWidget" name="ruleListLayoutWidget">
      <layout class="QVBoxLayout" name="ruleListLayout">
       <item>
        <layout class="QHBoxLayout" name="ruleListHeaderLayout">
         <item>
          <widget class="QLabel" name="ruleListLabel">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Download Rules</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="renameRuleBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Rename selected rule. You can also use the F2 hotkey to rename.</string>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeRuleBtn">
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="addRuleBtn">
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="ruleList">
         <property name="contextMenuPolicy">
          <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="ruleDefSplitter">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <widget class="QScrollArea" name="ruleScrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContentsOnFirstShow</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>370</width>
          <height>277</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,0,0,0,0,0,0,1">
         <item>
          <layout class="QHBoxLayout" name="priorityLayout">
           <item>
            <widget class="QLabel" name="priorityLabel">
             <property name="text">
              <string>Priority:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="prioritySpinBox"/>
           </item>
           <item>
            <spacer name="prioritySpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkRegex">
           <property name="text">
            <string>Use Regular Expressions</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="labelMustContain">
             <property name="text">
              <string>Must Contain:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineContains"/>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="labelMustStat">
             <property name="maximumSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelMustNotContain">
             <property name="text">
              <string>Must Not Contain:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineNotContains"/>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="labelMustNotStat">
             <property name="maximumSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lblEFilter">
             <property name="text">
              <string>Episode Filter:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEFilter"/>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="lblEFilterStat">
             <property name="maximumSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="labelEpFilterStat">
             <property name="maximumSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkSmart">
           <property name="toolTip">
            <string>Smart Episode Filter will check the episode number to prevent downloading of duplicates.
Supports the formats: S01E01, 1x1, 2017.12.31 and 31.12.2017 (Date formats also support - as a separator)</string>
           </property>
           <property name="text">
            <string>Use Smart Episode Filter</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="ignorePeriodLayout">
           <item>
            <widget class="QLabel" name="lblIgnoreDays">
             <property name="text">
              <string extracomment="... X days">Ignore Subsequent Matches for (0 to Disable)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinIgnorePeriod">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="specialValueText">
              <string>Disabled</string>
             </property>
             <property name="suffix">
              <string> days</string>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>365</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="lblLastMatch">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="torrentParametersGroupBox">
           <property name="title">
            <string>Torrent parameters</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="lblListFeeds">
          <property name="font">
           <font>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Apply Rule to Feeds:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listFeeds"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="matchingArticlesLayoutWidget">
      <layout class="QVBoxLayout" name="matchingArticlesLayout">
       <item>
        <widget class="QLabel" name="matchingArticlesLabel">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Matching RSS Articles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="matchingArticlesTree">
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <widget class="QPushButton" name="importBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>&amp;Import...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>&amp;Export...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
